conf/ocsci/flexible_scaling.yaml

```diff
@@ -0,0 +1,6 @@
+---
+# This configuration file is used for enable flexible scaling in the deployment
+# Valid only for OCS >= 4.7
+# https://issues.redhat.com/browse/KNIP-1544
+ENV_DATA:
+  enable_flexible_scaling: true
```

Comment:
Please describe this value in the documentation here:
https://github.com/red-hat-storage/ocs-ci/blob/master/conf/README.md

Reply:
I see you missed this one

Reply:
Done.